AdminCore.DatabaseMigration/Program.cs

```diff
@@ -12,7 +12,7 @@ public static class Program
     [ExcludeFromCodeCoverage]
     public static void Main(string[] args)
     {
-      DependencyInjection.RegisterDependencyInjection();
+      DependencyInjection.RegisterDependencyInjection(null, null);
```

Comment:
Can we overload this? Seems to be a lot of bulls passed into this method with usage